docs/_data/xpath_funs.yml

```diff
@@ -7,24 +7,93 @@ aliases:
     type: "xs:string"
     description: "The qualified name of a Java class, possibly with pairs of brackets to indicate an array type.
                   Can also be a primitive type name."
+  - &node_param
+    name: element
+    type: "xs:element"
+    description: "Any element node"
   - &needs_typenode "The context node must be a {% jdoc jast::TypeNode %}"
+  - &coord_fun_note |
+      The function is not context-dependent, but takes a node as its first parameter.
+      The function is only available in XPath 2.0.
```

Comment:
This probably needs to be rephrased for PMD 7 when merging... IIRC we have there 2.0 and 3.1 and in both the function would be available

Reply:
You're right, I'll prepare the merge